src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -28,31 +28,29 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
```

Comment:
@debermudez had you resolve a merge conflict in a rebase--feel free to re-review 🙏

Reply:
The main change is that this doc now uses `tensorrtllm` instead of `trtllm` for `--backend`.